modules/sdk/project-templates/project-templates-mvcbean-cdi-jsp-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,79 @@
+buildscript {
+	dependencies {
+		classpath group: "com.liferay", name: "com.liferay.css.builder", version: "@com.liferay.css.builder.version@"
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.css.builder", version: "@com.liferay.gradle.plugins.css.builder.version@"
+	}
+
+	repositories {
+		maven {
+			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+apply plugin: "com.liferay.css.builder"
+#if (${projectType} != "workspace")
+apply plugin: "war"
+#end
+
+dependencies {
+#if (${dependencyManagementEnabled} == "true")
+	compile (group: "org.hibernate.validator", name: "hibernate-validator") {
+		exclude group: "javax.validation", module: "validation-api"
+	}
+	compile (group: "org.hibernate.validator", name: "hibernate-validator-cdi") {
+		exclude group: "javax.validation", module: "validation-api"
+	}
+	compile group: "org.slf4j", name: "slf4j-api"
+
+	compileOnly group: "com.liferay", name: "com.liferay.bean.portlet.api"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib"
+	compileOnly group: "javax.mvc", name: "javax.mvc-api"
+	compileOnly group: "javax.portlet", name: "portlet-api"
+	compileOnly group: "javax.servlet", name: "javax.servlet-api"
+	compileOnly group: "javax.validation", name: "validation-api"
+	compileOnly group: "javax.ws.rs", name: "javax.ws.rs-api"
+	compileOnly group: "jstl", name: "jstl"
+
+	runtime group: "org.slf4j", name: "slf4j-log4j12"
+#else
+	compile (group: "org.hibernate.validator", name: "hibernate-validator", version: "6.0.14.Final") {
+		exclude group: "javax.validation", module: "validation-api"
+	}
+	compile (group: "org.hibernate.validator", name: "hibernate-validator-cdi", version: "6.0.14.Final") {
+		exclude group: "javax.validation", module: "validation-api"
+	}
+	compile group: "org.slf4j", name: "slf4j-api", version: "1.7.25"
+
+	compileOnly group: "com.liferay", name: "com.liferay.bean.portlet.api", version: "1.0.3"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "3.13.0"
```

Comment:
@gamerson The MVCBean templates will *only* work in 7.2 (and forward) since they depend on the MVC 1.0 implementation currently under review. They will not work in 7.0 or 7.1.

However the PortletMVC4Spring templates will work in 7.1 (and forward) since they depend on the Portlet 3.0 API.

Regarding the com.liferay.portal.kernel-3.13.0.jar version, that was the minimum version required to make things work in master (7.2).